app/helpers/application_helper.rb

```diff
@@ -22,11 +22,15 @@ def locale_for(type:, term:, record_class:)
     default_locale     = t("simple_form.#{type}.#{work_or_collection}.#{@term}").html_safe
     locale             = t("hyrax.#{@record_class}.#{type}.#{@term}").html_safe
 
-    return default_locale if locale.include?('translation missing')
+    return default_locale if missing_translation(locale)
 
     locale
   end
 
+  def missing_translation(value)
+    value.include?('translation missing')
+  end
```

Comment:
created this to use on "app/views/hyrax/admin/appearances/_color_input.html.erb". then refactored the above existing methods to use it as well.